### PR TITLE
Update GetVehiclePlateType.md

### PR DESCRIPTION
**Pull request by Kanzia-WOT**
Added returned values for reference of others

### CODE REVIEW

**Review by PsychoShock — APPROVED** [review]
By looking into decompiled files, I have only found this returning "1" so I don't think it can return this. I have done multiples tests with vanilla cars and none has return "1". All the rest is okay :)

image

**Comment by PsychoShock** [comment]
Note also that the native returns "0" when the player is not in a vehicle

**Comment by gottfriedleibniz** [comment]
Reference/use the enum stored in gamecode:

```c
enum eVehiclePlateType
{
	VPT_FRONT_AND_BACK_PLATES = 0,
	VPT_FRONT_PLATES = 1,
	VPT_BACK_PLATES = 2,
	VPT_NONE = 3,
};
```

Just ensure to wrap the above in codeblocks in the markdown.

File: VEHICLE/GetVehiclePlateType.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ ns: VEHICLE
 int GET_VEHICLE_PLATE_TYPE(Vehicle vehicle);
 ```
 
+```
```
Comment (stannum-cfx) [inline VEHICLE/GetVehiclePlateType.md:11]: Please remove the code block from the description. ([example template](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md#example-documentation))

**Review by stannum-cfx — CHANGES_REQUESTED** [review]

**Pull request merged**